src/migrations/1645105081399-QuestionnaireVersion.ts

```diff
@@ -0,0 +1,61 @@
+import { MigrationInterface } from 'typeorm';
+
+const MongoClient = require('mongodb').MongoClient;
+const configService = require('../config/config.service').configService;
+
+export class QuestionnaireVersion1645105081399 implements MigrationInterface {
+    public async up(): Promise<void> {
+        const client = new MongoClient(
+            configService.getMongoConnectionString(),
+            {
+                useUnifiedTopology: true,
+            },
+        );
+
+        try {
+            await client.connect();
+
+            const db = client.db();
+
+            await db.collection('questionnaire_versions').updateMany(
+                { language: { $exists: false } }, // Check if field exists
+                { $set: { language: 'en' } }, // Update field with a value
```

Comment:
We should retrieve language here from the current questionnaire and then update the language here